ui/nuxeo-document-preview.js

```diff
@@ -374,6 +376,14 @@ import './viewers/nuxeo-video-viewer.js';
     get _isVisible() {
       return Boolean(this.offsetWidth || this.offsetHeight);
     }
+
+    _videoHasCodecs(codec) {
```

Comment:
The name `_videoHas...` suggests that this would be a boolean, which is not true.